### ISSUE TITLE
Credentials on redirection are not being sent.

### ISSUE THREAD

**Issue by daniel-cues** [axios/axios#6126]
### Describe the bug

I'm running this on my frontend to make a request in my application

```
logout: async function() {
  	const response = await axios.get('/logout', {withCredentials: true});
  	return response.data;
  }
```

The issue is, `/logout` redirects to an external site and axios is not setting the credentials (cookies in this case) stored in the browser for the external site. CORS is configured correctly and the redirect works with `fetch()`, url shorteners and navigating to /logout directly. Why doesn't axis support proper authentication on redirections? It is a core HTTP functionality.

### To Reproduce

Run any request that returns a redirection code to an authenticated third party.

### Expected behavior

User gets redirected with proper credential management.

### Axios Version

0.19.0

### Browser

Firefox, Safari

### Node.js Version

12.22.12

### OS

MacOS, Windows, Linux

**Comment by mcblum** [comment]
Did you ever figure this out? I'm seeing the same issue!

**Comment by daniel-cues** [comment]
@mcblum nope, I removed Axios and used fetch instead. Why use an incomplete library that can’t even perform equally as good as the base code?

**Comment by mohammadazeemwani** [comment]
> @mcblum nope, I removed Axios and used fetch instead. Why use an incomplete library that can’t even perform equally as good as the base code?

regarding this can you please show the fetch request: 

my case is that i am using fetch to do request on a protected route on backend so i do { credentials: true } in fetch but if i do redirect from the backend it doesn't work else sending stringified objects work but redirects don't 

i have set cors on my backend server with origin: fronendDomain.  
Don't know why things are breaking.
